source/appendices/a-glossary-of-terms.rst

```diff
@@ -1009,12 +1001,10 @@ Note: The main Code of Practice document takes precedence in the case of any con
 
    * - Section
      - A group of chapters or articles.
-
-       A COUNTER Section_Type.
      -
 
    * - Section_Type
-     - A COUNTER report attribute that identifies the type of section that was accessed by the user.
+     - Defunct. A COUNTER R5 report attribute that identifies the type of section that was accessed by the user.
```

Comment:
Maybe add "R5.1" to clarify that this was a change between R5 and R5.1, and use past tense for R5 ("identified")?